### PR TITLE
Small fixes and tweaks

### PR DESCRIPTION
**Pull request by ThePiachu**
## About The Pull Request

Saw a few minor bugs and things that could be improved, so I fixed / tweaked them. Read the changelog.

For the Working Joe stuff - when I played them, it always took me a second or two to remember that "alwaysknow" is the way to invoke "the Working Joe line", so I added a secondary way to trigger that in a way that I associated it - `*workingjoe`. Same with the mess line. But apparently the parser likes to remove the final "s", so I added a backup way of invoking it, `*mes`.

## Changelog

It improves the experience of the game to make the errors less noticeable.

:cl:
fix: Mine boxes can now hold all the mines they are supposed to hold.
fix: Hydroponics locker now starts with the correct lock indicator.
spellcheck: Ravager's abilities now have proper capitalization.
qol: Some Working Joe emotes have an alternative way of invoking them now to help you embody the character easier.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -728,6 +735,16 @@
...
+		if("mess")
+			if(Joe && src.loc)
+				m_type = 1
+				playsound(src.loc, 'sound/voice/joe_awful.ogg', 75, 0)
+				say("Tut, tut. What an awful mess.")
+		if("mes") //Since the parser trims the final s...
+			if(Joe && src.loc)
+				m_type = 1
+				playsound(src.loc, 'sound/voice/joe_awful.ogg', 75, 0)
+				say("Tut, tut. What an awful mess.")
```
Comment (fira) [inline code/modules/mob/living/carbon/human/emote.dm:747]: This is really weird but if you absolutely need this, you can use `if("mess","mes")`

**Review by fira — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (ThePiachu) [reply code/modules/mob/living/carbon/human/emote.dm]: Ah neat, thanks for the tip, fixed!

**Review by fira — APPROVED** [review]
note the xeno verbs aren't supposed to be seen to begin with (as noted by hidden=1), they're just macro helpers

**Comment by ThePiachu** [comment]
Huh, that's strange, since I did see them in my game:

image

**Pull request merged**